extensions/aws/AWSLoader.h

```diff
@@ -16,43 +16,48 @@
  * limitations under the License.
  */
 
-#ifndef NIFI_MINIFI_CPP_AWSLOADER_H
-#define NIFI_MINIFI_CPP_AWSLOADER_H
+#pragma once
+
+#include <vector>
+#include <string>
+#include <memory>
 
 #include "core/ClassLoader.h"
 #include "utils/StringUtils.h"
 #include "controllerservices/AWSCredentialsService.h"
+#include "processors/PutS3Object.h"
 
 class AWSObjectFactory : public core::ObjectFactory {
-
  public:
   AWSObjectFactory() = default;
 
   /**
    * Gets the name of the object.
    * @return class name of processor
    */
-  virtual std::string getName() override{
+  std::string getName() override {
     return "AWSObjectFactory";
   }
 
-  virtual std::string getClassName() override{
+  std::string getClassName() override {
     return "AWSObjectFactory";
   }
 
   /**
    * Gets the class name for the object
    * @return class name for the processor.
    */
-  virtual std::vector<std::string> getClassNames() override{
+  std::vector<std::string> getClassNames() override {
     std::vector<std::string> class_names;
     class_names.push_back("AWSCredentialsService");
```

Comment:
don't we need to add `PutS3Object` to the `class_names` vector as well?

Reply:
I suppose we do, I will have to check how it worked without it even in EFM. Fixed in [d2fc8cd](https://github.com/apache/nifi-minifi-cpp/pull/915/commits/d2fc8cdf064177ec7188ff4e80228724c1711fb2)